Jetcaster/app/src/main/java/com/example/jetcaster/util/ViewModel.kt

```diff
@@ -31,7 +31,8 @@ fun <VM : ViewModel> viewModelProviderFactoryOf(
 ): ViewModelProvider.Factory = SimpleFactory(create)
 
 /**
- * This needs to be a named class currently to workaround a compiler issue: b/163807311
+ * This needs to be a named class currently to workaround a compiler issue:
+ *  https://issuetracker.google.com/163807311
```

Comment:
I get access denied for this. Is it public?

Reply:
Doesn't seem so. I'd revert.

Reply:
Done